src/scripts.js

```diff
@@ -1 +1,42 @@
-console.log("Hello World");
\ No newline at end of file
+const newUserRepo = new UserRepository(userData);
+const newUser = new User(userData[0]);
+
+
+/* *****Query Selectors***** */
+const welcomeMessage = document.querySelector("#welcomeMessage");
+const infoCard = document.querySelector("#infoCard");
+const stepGoals = document.querySelector("#stepGoals");
+
+/* *****Event Listeners***** */
+window.addEventListener("load", displayUser);
+
+
+/* *****Functions***** */
+function displayUser() {
+  welcomeUser();
+  displayInfoCard();
+  compareStepGoal();
+}
+
+function welcomeUser() {
+  welcomeMessage.innerText = `Welcome, ${newUser.firstName()}!`;
+}
+
+function displayInfoCard() {
```

Comment:
Do we need to include all the user detail on the card? I'll style it initially with all this data but let's look at it all together when the styling is complete